test/fixtures/projects/helpers/ParseOptionsData.ts

```diff
@@ -16,37 +16,30 @@ export const initValues: ParseOptionValues = {
 
 export const initSchema: ParseOptionSchema[] = [{
   name: 'delimiter',
-  label: 'Delimiter',
```

Comment:
I see this file being required by the ["Importer.ts" fixture](https://github.com/gristlabs/grist-core/blob/a375620bb8f447581641e5ee86c9e35a52883131/test/fixtures/projects/Importer.ts).

But I can't see any test using it:
```bash
$ fgrep -R "/Importer" test/
test/fixtures/projects/Importer.ts:import {Importer, SourceInfo} from 'app/client/components/Importer';
```

Reply:
@georgegevoian tracked this down to a commit in grist-saas from 2019. Comment was:
> Added fixtures for Importer and ParseOptions to test/fixtures/projects/, to make UI styles easier and faster to inspect manually.

This class of test got moved to grist-core in https://github.com/gristlabs/grist-core/commit/d07cb3d47568552c14b87e4219c3cc984ab93b1a. But it doesn't seem to have been added to CI, so I think https://github.com/gristlabs/grist-core/commit/b87c91cd92c54aff12693f6bdaf32521fe2200a6 perhaps broke it with a version change because I can't run the tests (cc @berhalak).

Reply:
It looks like a dead code, but it is also possible that I removed the test that was using this fixture when I moved all tests to grist-core. Will investigate it, thanks for catching it.